annotations/templates/annotations/relations.html

```diff
@@ -18,7 +18,7 @@
         <div class="col-xs-4  clearfix text-center">
             {% if relations.has_previous  %}
             <div class="pull-left">
-                <a href="?page={{ relations.previous_page_number }}" class="btn btn-xs">&laquo; Previous</a>
+                <a href="?page={{ relations.previous_page_number }}{% if not createdBy %}{% else %}&createdBy={{ createdBy }}{% endif %}{% if not project %}{% else %}&project={{ project }}{% endif %}{% if not createdBefore %}{% else %}&createdBefore={{ createdBefore }}{% endif %}{% if not createdAfter %}{% else %}&createdAfter={{ createdAfter }}{% endif %}{% if not occursIn %}{% else %}&occursIn={{ occursIn }}{% endif %}{% if not terminal_nodes %}{% else %}&terminal_nodes={{ terminal_nodes }}{% endif %}" class="btn btn-xs">&laquo; Previous</a>
```

Comment:
besides the comment below (try to use the filter object), why the empty if statement???
```
{% if not createdBy %}{% else %}&createdBy={{ createdBy }}{% endif %}
```
just unnecessary code (why not testing for the opposite with no else statement?